.gitignore

```diff
@@ -1,20 +1,8 @@
-# We filter only autotools artefacts.
 # Use a separate build directory!
+# Have a look at https://wiki.gnucash.org/wiki/Documentation_Improvement#Initial_Steps
 # For other purposes adjust "$XDG_CONFIG_HOME/git/ignore!
```

Comment:
Perhaps clearer:
If /your/ tools add files into the source tree, adjust /your/ "$XDG_CONFIG_HOME/git/ignore" file instead.

Reply:
https://git-scm.com/docs/gitignore has the full description of the possibilities.